### PR TITLE
Synacormedia bid adapter: add support for floors module

### PR DESCRIPTION
**Pull request by rajcspsg**
## Type of change

- [ x] Feature

## Description of change

Added support for floors module

Be sure to test the integration with your adserver using the [Hello World](/integrationExamples/gpt/hello_world.html) sample page.

- contact email of the adapter’s maintainer
- [ ] official adapter submission
   email = eng-demand@imds.tv

### CODE REVIEW

**Comment by ChrisHuie** [comment]
@rajcspsg looks like there a few linting issues still in this pr if you could look at them when you get a chance :)

File: modules/synacormediaBidAdapter.js [hunk 1]
```diff
@@ -68,10 +68,6 @@ export const spec = {
         seatId = bid.params.seatId;
       }
       const tagIdOrplacementId = bid.params.tagId || bid.params.placementId;
```
Comment (musikele) [inline modules/synacormediaBidAdapter.js:70]: The variable name is still `tagIdOrplacementId` (note the lower case `p`), while later it has been corrected to `tagIdOrPlacementId` (upper-case `P`).

**Review by musikele — CHANGES_REQUESTED** [review]
Please check this error - you can also view in console by launching `npx gulp lint`.

File: modules/synacormediaBidAdapter.js (hunk 1 above)
Comment (rajcspsg) [reply modules/synacormediaBidAdapter.js]: @musikele I fixed the typo

**Comment by rajcspsg** [comment]
> @rajcspsg looks like there a few linting issues still in this pr if you could look at them when you get a chance :)

I couldn't find any issue with linting. I did `gulp lint`

**Review by musikele — APPROVED** [review]
LGTM

**Pull request merged**